package.json

```diff
@@ -72,7 +72,7 @@
     "registry": "https://registry.npmjs.org"
   },
   "volta": {
-    "node": "14.19.3",
```

Comment:
had to do this to tell yarn to install the dependencies of `@embroider/unstable-release`. Not sure if changing the development node is a good idea, or if there is a way to allow yarn to ignore the root node version some how.